config/400-config-br-default-channel.yaml

```diff
@@ -1 +1 @@
-core/configmaps/config-br-default-channel.yaml
\ No newline at end of file
+core/configmaps/br-default-channel.yaml
```

Comment:
Is this change unrelated?

Reply:
Done, updated the PR description. Thanks!